src/Microsoft.IdentityModel.Tokens/Validation/Results/Details/AudienceValidationError.cs

```diff
@@ -47,7 +47,9 @@ protected override Exception CreateException()
         {
             if (ExceptionType == typeof(SecurityTokenInvalidAudienceException))
             {
-                var exception = new SecurityTokenInvalidAudienceException(MessageDetail.Message, InnerException) { InvalidAudience = Utility.SerializeAsSingleCommaDelimitedString(TokenAudiences) };
+                var exception = TokenAudiences != null ?
```

Comment:
I don't understand this change, @pmaytak. Is it related to the fix? or some nit?
Oh it is, because the exception will be built with nulls. line 108. Might have needed a little comment to help the reviewers :-)
Got it.

Reply:
Yes, this nit is only for the new model, since this would get serialized as "null".